scripts/pre_commit_linter.py

```diff
@@ -474,6 +708,7 @@ def _lint_css_files(
         files_to_lint: list(str). A list of filepaths to lint.
         stdout:  multiprocessing.Queue. A queue to store Stylelint outputs.
         result: multiprocessing.Queue. A queue to put results of test.
+        mode: str. Mode i.e. verbose or non-verbose.
```

Comment:
This is really a boolean, isn't it? If so, make it so, here and elsewhere.

Reply:
Done